Storage/Storage/CoreData/CoreDataManager.swift

```diff
@@ -81,62 +81,20 @@ public final class CoreDataManager: StorageManagerType {
         return backgroundContext
     }()
 
+    private let queue = DispatchQueue(label: "persistentContainerQueue")
```

Comment:
These changes will be dropped from the branch before merging the PR, they are needed now until the PR #14533 is merged.